mkosi/__init__.py

```diff
@@ -3259,9 +3259,11 @@ def make_image(
                 context.config,
                 cmdline=cmdline,
                 options=opts,
-                certificate=context.config.verity_certificate if verity else None,
+                certificate=context.config.verity_certificate
+                if verity in (Verity.auto, Verity.signed)
+                else None,
```

Comment:
Ugh, this looks like another ruff edge case. Can we make this
```suggestion
                certificate=(
                    context.config.verity_certificate if verity in (Verity.auto, Verity.signed) else None
                ),
```